src/api-docs.js

```diff
@@ -3843,4 +3839,31 @@ const apiDoc = {
   },
 };
 
-export default apiDoc;
+/**
+ * Generates the Api Docs according to the current configurations of the Headless Wallet
+ */
+async function getApiDocs() {
```

Comment:
Why is it `async`?

Reply:
Since the `getApiDocs()` is expected to fetch the latest configurations for the headless instance, I expect it to eventually grow into a promise, needing to fetch information from some other asynchronous source. I added the `async` as a way to keep it already future-proof in that sense.

However, it's not a needed feature at the moment and can safely be refactored to a synchronous method.

Reply:
`getApiDocs()` reverted from an `async` function to a regular one on [6f75250](https://github.com/HathorNetwork/hathor-wallet-headless/pull/327/commits/6f75250c7ddb85167f520c4f0ea480a53e283bfa)